app/Console/Kernel.php

```diff
@@ -43,7 +43,7 @@ protected function schedule(Schedule $schedule)
         $schedule->job(new SandboxCleanupJob)->everyFifteenMinutes();
 
         // Schedule site stat updates for each wiki and platform-summary
-        $schedule->command('schedule:stats')->daily();
+        $schedule->command('schedule:stats')->dailyAt('7:00');
```

Comment:
This triggers jobs for both platform summary and per site stats: https://github.com/wbstack/api/blob/e9546668cd92f937bebf0885ab700ac6cfda9f78/app/Console/Commands/ScheduleStatsUpdates.php#L54-L57